### PR TITLE
adds the box gun [dnm] [wip] [worldstar] 

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: adds a gun made from a box, a modular receiver, 5 cable coils and 2 metal rods
capture

:cl: zamolxius
add: Added box gun, an ancient part of the grayshit heritage, now unearthed for the modern specimens of hippie station. It can shoot any bullet, but may explode on firing, taking one of your arms with it to the shadow realm
imageadd: added a new sprite, for the box gun
/:cl:

[why]: 
**+can shoot any bullet (40mm grenades don't work)
+easy to make if you get your hands on a modular receiver
+looks like a box
-only holds one single bullet, must be loaded after each shot
-will explode 10% of the time you shoot it, removing one of your arms**

also it's pretty much the goon zipgun iirc

### CODE REVIEW

**Comment by k-k-karma** [comment]
this is basically the contender (a traitor item) with downsides

**Comment by ghost** [comment]
so what's your point dude, it's still way way weaker than the contender, shoots only one bullet ffs, which in 1/10 cases will explode your arm off (feedback appreciated, i can make it more likely to explode).

going by your logic the space suits at the pods are basically the syndicate space suits, or the balisong is basically the energy syndicate balisong (a traitor item) with downsides, we also got the improvised crossbow which works exactly like the former engineering traitor item we had, the rod gun, impaling people next to walls, basically a traitor item with downsides, it only has less shots and different loading mechanic

**Comment by k-k-karma** [comment]
you're giving an 8tc traitor item to greytiders

**Comment by ghost** [comment]
let's be serious this item wouldn't be worth 2tc especially because you need ammo in the first place, and it blows your damn arm off with an explosion that also blinds you for a critical moment, not to mention it only holds 1 bullet
you've been pissy with me all the while on discord, what's your problem karma?

**Comment by k-k-karma** [comment]
just stating facts

**Comment by mikolaj007** [comment]
unless i got some real nice ammo to shoot, this gun sucks.
i say, sure. why not.

**Comment by AnthonyJNgo** [comment]
Why would anyone be inclined to make or use this? I can't think of a single scenario where I'd eant to fire one bullet at the risk of losing my arm and probably going into crit.

**Comment by Manly-Dwarf** [comment]
The thing with a zip gun on goon being able to hold any ammo means ANY ammo, and goon ammunition is unbelievably powerful. You can make a ghetto dets revolver, you can make an 80 brute + stun buckshot launcher, even rocket launcher ammo can be used.
It's not really the same, and we have the contender any way. Also sprite is 0/10.

**Comment by ghost** [comment]
the energy butterfly knife is worth 15tc, and there's a civilian version available which still does quite a lot of damage, the difference between them is way smaller than between the contender (8tc) and this gun.

also @AnthonyJNgo , because it gives you a ranged option, you could make a spear and risk going melee, or you can do this

**Comment by mikolaj007** [comment]
id throw a grenade spear or something.
whats a good one shot bullet thats maints-salvagable?

**Comment by ghost** [comment]
@kawaii-big-boss , it can't hold grenade launcher shells, plus it's the same thing as a shortened shotgun for a huge risk. is it alright if I make it 1/5 or 1/4 chance to blow up? contender is a traitor item and more cool improvised and risky to use shit is always welcomed

**Comment by mikolaj007** [comment]
if you make it 1/4 or 1/5, then it's double useless

**Comment by RafRoq** [comment]
would someone actualy take it ?

**Comment by ghost** [comment]
also the sprite is a box with a modular receiver and a makeshift handle from 2 rods with a few cables, dont think anything more is expected from an improvised weapon

**Comment by k-k-karma** [comment]
it can use grenade shells from a quick glance

**Comment by Manly-Dwarf** [comment]
"energy butterfly knife is worth 15TC"
8 TC.
"civilian version which still does quite a lot of damage"
30 brute, only on the back, doesn't work when they're stunned. No. The 10 brute is easy to replicate with literally anything.

Really, this gun would be a lot better if we had real gunsmithing, so you could actually make bullets that this thing could hold. All-caliber's current state is a death sentence for this thing being a viable feature, since there's "weak as shit" and "stupidly good" on top of having to consider the Contender. Lethals right now are in such an odd spot right now.

**Comment by ghost** [comment]
grenade shells can be shot but only activate when shot by a grenade launcher

**Comment by k-k-karma** [comment]
so it's bugged and you cant be bothered to fix it

**Comment by ghost** [comment]
> "energy butterfly knife is worth 15TC"
> 8 TC.
> "civilian version which still does quite a lot of damage"
> 30 brute, only on the back, doesn't work when they're stunned. No. The 10 brute is easy to replicate with literally anything.
> 
> Really, this gun would be a lot better if we had real gunsmithing, so you could actually make bullets that this thing could hold. All-caliber's current state is a death sentence for this thing being a viable feature, since there's "weak as shit" and "stupidly good" on top of having to consider the Contender. Lethals right now are in such an odd spot right now.

yes, but all those "stupidly good" ammo requires additional work, hacking the lathe, getting metal. it's practically a quirky improvised shotgun that way



> so it's bugged and you cant be bothered to fix it

it's not a bug, it's by design so you can't use them in the contender. do you have something with me, karma? what's your deal? why such a jerk today?

**Comment by k-k-karma** [comment]
because it's just A FUCKING RESKIN OF THE CONTENDER WITH MINIMAL FUCKING EFFORT

**Comment by Manly-Dwarf** [comment]
Hacking the lathe and getting metal takes two seconds. EVA is legit just down the hall from cargo.

**Comment by ghost** [comment]
> because it's just A FUCKING RESKIN OF THE CONTENDER WITH MINIMAL FUCKING EFFORT

what is the balisong then?
also it's a legit improvised weapon with it's own niche, that being ranged craftable weapon. we have spears and tomahawks for melee, we have IED for explosives, we have this and the improvised shotgun for ranged


> Hacking the lathe and getting metal takes two seconds. EVA is legit just down the hall from cargo.


which isn't a bad thing because the only "op" ammo would be shotgun shells, which you can use anyways with an improvised sawed off shotgun, accomplishing the same with with less risk of getting your arm blown off

**Comment by Manly-Dwarf** [comment]
So if the sawed off shotgun takes only an additional ten seconds to make, why would you bother with the box gun? What's the purpose?

**Comment by ghost** [comment]
it looks like a box when held inhand, it can take any bullet for additional versatility

**Comment by k-k-karma** [comment]
non-obvious inhands dont matter much cos people'll get used to it

**Comment by ghost** [comment]
that's not an argument since people will get used to anything, it is still less noticeable than most other ranged weapons

**Comment by danhibiki** [comment]
Make box gun only really usable if you have clumsy, otherwise you should have at least a 50% chance of boning yourself

**Comment by mikolaj007** [comment]
if you want a non-obvious inhand, just use a package wrapper.
z>left click on the target and they wont have time to react

**Comment by k-k-karma** [comment]
delete that comment newfags cant know

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -227,4 +227,34 @@
...
+	icon_state = "box_gun"
+	item_state = "syringe_kit"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/box_gun
+	recoil = 1
+
+/obj/item/gun/ballistic/revolver/doublebarrel/contender/box_gun/afterattack(atom/target, mob/living/user, flag, params)
+	if(istype(user) && prob(10))
+		var/obj/item/bodypart/l_arm = user.get_bodypart(BODY_ZONE_L_ARM)
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/weaponry.dm:245]: l_arm is an already existing object same goes with r_arm?

Comment (k-k-karma) [reply hippiestation/code/game/objects/items/weaponry.dm]: ?

**Comment by zamolxius** [comment]
oh?

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/weaponry.dm:245]: dont know how else to chop an arm off

**Comment by WarRage333** [comment]
This cant come into the game as long forge makes bullets

10% is also too low make it 33%

**Comment by ghost** [comment]
oh shit good point about the forge,  maybe i could blacklist it or make it so using forge ammo just always explodes the gun

**Comment by WarRage333** [comment]
I don't really oppose this that much but if this gets merged buff contender ammo capacity

**Comment by GuyonBroadway** [comment]
This thing could fire mercury .50 cal rounds or (god help you) gold .50 cal rounds.

Gold .50cal can kill a man stone dead if you hit them right.

**Comment by GuyonBroadway** [comment]
Forgot to mention, you can make those bullets in the arc forge with minimal effort

**Pull request closed without merging**